File: app/src/main/java/tutorials/acviewmodel/networking/RepoService.java
package tutorials.acviewmodel.networking;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;
import tutorials.acviewmodel.model.Repo;

public interface RepoService {

    @GET("orgs/Google/repos")
    Call<List<Repo>> getRepositories();

    @GET("repos/{owner}/{name}")
    Call<Repo> getRepo(@Path("owner") String repoOwner, @Path("name") String repoName);

}

File: app/src/main/java/technologies/acviewmodel/networking/RepoService.kt
package technologies.acviewmodel.networking

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Path
import technologies.acviewmodel.model.Repo

interface RepoService {

    @get:GET("orgs/Google/repos")
    val repositories: Call<List<Repo>>

    @GET("repos/{owner}/{name}")
    fun getRepo(@Path("owner") repoOwner: String, @Path("name") repoName: String): Call<Repo>

}
